cmd/kava/opendb/run.sh

```diff
@@ -0,0 +1,3 @@
+CGO_CFLAGS="-I/opt/homebrew/opt/rocksdb/include" \
+CGO_LDFLAGS="-L/opt/homebrew/opt/rocksdb/lib -lrocksdb -lstdc++ -lm -lz -L/opt/homebrew/opt/snappy/lib -L/opt/homebrew/opt/lz4/lib -L/opt/homebrew/opt/zstd/lib" \
+  go test -v -tags=rocksdb
```

Comment:
i think this would be better added to the root dir's makefile which handles cross platform CGO flags and is already included in CI pipelines

Reply:
I agree

Reply:
fixed